vendor-onboarding-checklist.md

```diff
@@ -0,0 +1,175 @@
+Information System Replacement (EIS-R) Project. Specifically, this
+document addresses our agile acquisition practice and onboarding new
+vendor staff. Within agile acquisition, we anticipate bringing in a
+potentially different vendor every three to four months, possibly more
+often when we begin running multiple acquisitions simultaneously. With
+short timelines, this onboarding checklist will bring new vendor staff
+up to speed as quickly as possible.
+
+# Audience
+
+This document intends to inform EIS-R State of Alaska project staff and
+the vendors we engage with in various product-increment acquisitions.
+
+# Roles
+
+This section identifies the anticipated vendor roles that will be
+engaged with the
+project.
+
+| **Role**                 | **Description**                                                                             |
+| ------------------------ | ------------------------------------------------------------------------------------------- |
+| Project manager          | Manage the project                                                                          |
+| Technical lead           | Lead the technical work                                                                     |
+| Developers               | Develop client and server application components using CSS, HTML5, Javascript and .NET Core |
+| Researcher               |                                                                                             |
+| User experience designer | Design the user experience                                                                  |
+| Visual designer          |                                                                                             |
+| Scrum Master             | Lead internal daily scrum sessions                                                          |
+
+# Tools and resources
+
+This section identifies the tools and resources that vendors will need
+access to work with and what type of access they will need.
+
+<table>
+<thead>
+<tr class="header">
+<th><strong>Role</strong></th>
+<th><strong>Tools and Resources</strong></th>
+<th><strong>Able to</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>Project manager</td>
+<td><p>State of Alaska LDAP account</p>
+<p>State of Alaska VPN</p>
+<p>DHSS domain account</p>
+<p>DHSS SharePoint</p>
+<p>Visual Studio Team Services</p>
+<p>Trello</p>
+<p>Slack</p>
+<p>Alaska DHSS GitHub</p>
+<p>Mural</p></td>
+<td><ul>
+<li><p>Access Alaska WAN</p></li>
+<li><p>Access DHSS domain secured resources</p></li>
+<li><p>CRU docs on SharePoint</p></li>
+<li><p>Read and update work-items and PR comments in VSTS</p></li>
+<li><p>Read and update Trello</p></li>
+<li><p>Read and post messages to Slack</p></li>
+<li><p>Read GitHub repositories and issues</p></li>
+<li><p>Participate in Mural sprint retrospectives</p></li>
+</ul></td>
+</tr>
+<tr class="even">
+<td>Technical lead</td>

```

Comment:
@ctro Can you take a look at the list of resources the tech lead and dev's will need access to and provide some feedback?

Reply:
For the "Able to" section should we include specific abilities in VSTS? 

- abilities in the "Code" section; they can probably have pretty full reign except on `staging` and `master` branches.
- abilities in the "Build" section; can they edit build definitions? 
- abilities in the "Release" section; can they queue new releases?  edit release definitions?

That kinda takes me to Azure access/abilities. Should we add Azure? We plan on having 3 AppService Slots to begin with: prod, staging, dev.  can these people manually deploy to the development slot?  can they access the Kudu dashboard for Azure dev slot?

Reply:
We should chat about this stuff for sure.  I don't know what "full reign" in the Code section means.  What I imagine is that they can push new branches, create PRs.  What else would they need to do?  E.g., I wouldn't want them to change branch policies.  

I'm starting to think that I need to reformat the doc and pull it out of table format.  My tendency as you say things like "full reign" or "include specific abilities in VSTS" is to start listing out the sections (e.g., "Code") and subsections (e.g., "Files", "Commits", "Pushes", "Branches", "Tags", "Pull requests") and then identify what they can do.

On the other hand, I think what we're running into is that we don't have the roles defined yet.  So, there's a more detailed exercise of going through VSTS and thinking about "Does a tech lead need to do X?" or "Does a dev need to do X".  And this feels like a necessary exercise, but a totally different document from the "vendor-onboarding-checklist".  Thoughts?